v2/pages/userInstanceQuestionExam/userInstanceQuestionExam.js

```diff
@@ -90,6 +91,27 @@ router.get('/', function(req, res, next) {
                 callback(null);
             });
         },
+        // We can probably combine the previous function with the following one, and eliminate sql.get_submission
+        function(callback) {
+            res.locals.showAllSubmissions = false;
+            submissionFilePath = path.join(res.locals.course.path, "questions", res.locals.question.directory, "submission.html");
```

Comment:
We shouldn't know anything about the internal structure of questions inside this file. Instead just unconditionally get all the past submission data and hand it off to the question renderer.